pytensor/tensor/conv/abstract_conv.py

```diff
@@ -1886,9 +1886,7 @@ def frac_bilinear_upsampling(input, frac_ratio):
     pad = double_pad // 2
 
     # build pyramidal kernel
-    kern = bilinear_kernel_2D(ratio=ratio)[np.newaxis, np.newaxis, :, :].astype(
-        config.floatX
-    )
+    kern = bilinear_kernel_2D(ratio=ratio)[None, None, :, :].astype(config.floatX)
```

Comment:
I didn't mean you should remove all internal uses of `np.newaxis`, just the code I referenced in the comment